readme.md

```diff
@@ -38,13 +81,25 @@ board.on("ready", function() {
     "violet",
   ];
 
-  setInterval(() => {
-    pixels.forEach(pixel => pixel.color(colors[index]));
```

Comment:
In our case it's website (browserified johnny-five) <> Chrome app <> serial, and (eventually) also an Electron Chrome Standalone App, but no reason this can't still be ES6.